react-validated-input-component/src/index.jsx

```diff
@@ -0,0 +1,26 @@
+import React, { useState } from 'react';
+import reactDOM from 'react-dom/client';
+const container = document.querySelector('#root');
+const root = reactDOM.createRoot(container);
+
+function PasswordField() {
+  const [password, setPassword] = useState('');
+  if (password.length >= 8) {
+    var isGoodEnough = !!password;
+  }
+
+  const handlePassword = (event) => {
+    setPassword(event.target.value);
+  };
+  const iconClass = isGoodEnough ? 'fa-check' : 'fa-xmark';
+  return (
+    <div>
+      <input type="password" onChange={handlePassword} value={password}></input>
+      <i className={`fa-sharp fa-solid ${iconClass}`}></i>
```

Comment:
You've shown the icon, but you also need to add an error message below the input, as shown in the exercise.